src/Core/Domain/Product/Exception/BulkDeleteProductException.php

```diff
@@ -0,0 +1,63 @@
+<?php
```

Comment:
I don't see where this class is used 🤔

Reply:
@sowbiba Needs an answer

Reply:
It's not used and I cant remember why I added this class, so it's removed